docs/latest/design/extending-envoy-gateway.md

```diff
@@ -144,6 +144,62 @@ latency between Envoy and the authentication service and also allows the end use
 
 ![Control Plane Extension Example](../images/extension-auth-example.png)
 
+## Design Decisions
+
+- Since an HTTPRoute resource corresponds to an [Envoy Proxy Route resource][], the name of the Route is set using the name and namespace of the HTTPRoute that generated it so that any extension looking at the
```

Comment:
Is this how it is today or is this part of our proposal on changes that need to be made to EG? If it is the ladder it might be good to give some context by stating what are the challenges and then outline this as the proposed solution to address that challenge.

Reply:
Debating removing this line since i don't think its all that relevant anymore given that we don't pass any context in on the Listener hook. I know we don't need to, but perhaps we should?

Reply:
I think we should remove it, especially if we don't intend on actually doing the work. In this scenario, it is better for a user to use the route hook.